### DISCUSSION TITLE
Admin Issue - Relational field Grid not load

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#11567] [Q&A / Support]
Hello,

I am working on pimcore v10.3.1 and facing an issue on loading the grid of the relational field.
![image](https://user-images.githubusercontent.com/52573912/156345095-681245c7-2193-48ad-8b4d-7788cbfa5d22.png)

can anyone help me with how to sort out it?

**Comment by shubhangi92** [comment]
I am also facing similar issue but with other role not with admin . Can anyone help with this.This issue i am facing with version 10.1.3

**Reply by ghost** [reply]
@shubhangi92 try with this command, It's works for me

bin/console pimcore:search-backend-reindex
